src/category_theory/limits/limits.lean

```diff
@@ -133,6 +131,85 @@ def is_limit_iso_unique_cone_morphism {t : cone F} :
   { lift := λ s, (h s).default.hom,
     uniq' := λ s f w, congr_arg cone_morphism.hom ((h s).uniq ⟨f, w⟩) } }
 
+namespace of_nat_iso
+variables {X : C} (h : yoneda.obj X ≅ F.cones)
+
+/-- If `F.cones` is represented by `X`, each morphism `f : Y ⟶ X` gives a cone with cone point `Y`. -/
+def cone_of_hom {Y : C} (f : Y ⟶ X) : cone F :=
+{ X := Y, π := h.hom.app (op Y) f }
+
+/-- If `F.cones` is represented by `X`, each cone `s` gives a morphism `s.X ⟶ X`. -/
+def hom_of_cone (s : cone F) : s.X ⟶ X := h.inv.app (op s.X) s.π
```

Comment:
Not for this PR, but sometimes I wonder if `s.tip` is more readable? I occasionally stumble when reading `s.X → X`.

Reply:
I agree renaming here would be good, and easy to implement if we could pick names. I've heard people use `summit` and `nadir` (but which way round?!), but `tip` is fine too.